test/test_utils/sagemaker.py

```diff
@@ -185,8 +185,6 @@ def install_sm_local_dependencies(framework, job_type, image, ec2_conn):
     if framework == "tensorflow" and job_type == "inference":
         # TF inference test fail if run as soon as instance boots, even after health check pass. rootcause:
         # sockets?/nginx startup?/?
-        print("sleep 300s for tensorflow inference images to avoid socket issues")
```

Comment:
I think @Satish615 had added this line to avoid Socket Failures.

Reply:
This line itself causes socket error as the sleep time is 300 sec

Reply:
Thanks for solving the issue.